src/app/features/notification/notification-settings/notification-settings.component.scss

```diff
@@ -0,0 +1,133 @@
+@use "variables/colors";
+@use "../../../../styles/variables/breakpoints";
+
+.full-width {
+  width: 100%;
+}
+
+.notifications-container {
+  padding: 16px;
+}
+
+.notification-types-container {
+  margin-top: 30px;
+}
+
+.browser-notifications-container {
+  margin-top: 32px;
+}
+
+.notification-setting-title {
+  margin-bottom: 10px;
+}
```

Comment:
Why is this sometimes 30px sometimes 32px? We should probably work with our standard sizes from `@use "variables/sizes";`

Reply:
Thanks, I update this code according to your suggestion.